overlord/snapstate/backend/link_test.go

```diff
@@ -118,12 +118,29 @@ version: 1.0
 environment:
  KEY: value
 
+slots:
+  system-slot:
+    interface: dbus
+    bus: system
+    name: org.example.System
+  session-slot:
+    interface: dbus
+    bus: session
+    name: org.example.Session
+
 apps:
  bin:
    command: bin
  svc:
    command: svc
    daemon: simple
+ dbus-system:
+   daemon: simple
```

Comment:
There's no requirement that an activatable service use `daemon: dbus`.  It is usually a good idea, since it provides notification that the daemon has started without the extra privileges `daemon: notify` requires.

On the `dbus-daemon` side, it will issue the method call as soon as the bus name has been acquired, which could be before or after systemd thinks the daemon has started.